### PR TITLE
5.1 K8S and sidecar update and 5.0 unsupport

### PR DESCRIPTION
**Pull request by GunaKKIBM**

### CODE REVIEW

**Review by prankulmahajan — APPROVED** [review]
lgtm

**Review by ashimagarg27 — APPROVED** [review]

File: config-templates/ibm/ibm-vpc-block-csi-driver/changelog.json [hunk 1]
```diff
@@ -3,55 +3,35 @@
...
+                "golang": "1.20.11",
+                "ubi": "8.9-1108",
+                "new-features": [
+                    "Updated Kubernetes dependency to 1.28",
+                    "Updated csi node driver registrar image to v2.9.3",
+                    "Updated livenessprobe image to v2.11.0",
+                    "Updated csi-snapshotter image to v6.3.3",
+                    "Updated csi-provisioner image to v3.6.3",
+                    "Updated csi-attacher image to v4.4.3",
+                    "Updated csi-resizer image to v1.9.3"
```
Comment (mssachan) [inline config-templates/ibm/ibm-vpc-block-csi-driver/changelog.json:19]: should these come under `fixes` or `new features`?

Comment (mssachan) [reply config-templates/ibm/ibm-vpc-block-csi-driver/changelog.json:19]: Also even if `fixes` field is empty, dont remove it. Doc team asked to retain fields with even empty values.

File: config-templates/ibm/ibm-vpc-block-csi-driver/changelog.json [hunk 2]
```diff
@@ -3,55 +3,36 @@
         "5.1":{
             "status": "supported",
             "latest-revision": {
+                "revision": "3",
+                "release-date": "12 February 2014",
```
Comment (mssachan) [inline config-templates/ibm/ibm-vpc-block-csi-driver/changelog.json]: @GunaKKIBM If I remember correctly, I think we concluded release date as `16 February` based on input from @gayathrimenath-ibm

**Review by mssachan — APPROVED** [review]
lgtm

**Pull request merged**